config-vm.md

```diff
@@ -0,0 +1,49 @@
+# <a name="VirtualMachineSpecificContainerConfiguration" /> Virtual-machine-specific Container Configuration
+
+Virtual-machine-based containers require additional configuration to that specified in the [default spec configuration](config.md).
```

Comment:
`config.md` isn't the “default” configuration, it's the (mostly) platform-agnostic configuration.  It's not entirely platform-agnostic (e.g. `hooks` is POSIX-only), but I think we can do better than “default”.  Maybe “base”?  “main”?

Reply:
Done.